### DISCUSSION TITLE
Welcome to Ampersand Discussions!

### DISCUSSION THREAD

**Discussion by Michiel-s** [AmpersandTarski/Ampersand#1234] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by stefjoosten** [comment]
This is a test for e-mail to @stefjoosten

**Comment by hanjoosten** [comment]
Bij mij komt de email aan. Test geslaagd?

Op wo 23 nov. 2022 07:44 schreef Stef Joosten ***@***.***>:

**Comment by stefjoosten** [comment]
Nee, niet geslaagd. Ik had gehoopt dat ik de mail zou krijgen.

Weet jij onder welk e-mail adres mensen mij kunnen bereiken op GitHub?
Ik zou verwachten dat zoiets als ***@***.******@***.***> of ***@***.*** zou moeten werken, maar ho maar!

Ik vraag me af of zoiets wel bestaat: bereikbaar zijn onder een github e-mailadres.



From: Han Joosten ***@***.***>
Date: Wednesday, 23 November 2022 at 08:36
To: AmpersandTarski/Ampersand ***@***.***>
Cc: Joosten, Stef ***@***.***>, Mention ***@***.***>
Subject: Re: [AmpersandTarski/Ampersand] Welcome to Ampersand Discussions! (Discussion #1234)
Bij mij komt de email aan. Test geslaagd?

Op wo 23 nov. 2022 07:44 schreef Stef Joosten ***@***.***>:

**Comment by hanjoosten** [comment]
Het is allemaal een kwestie van hoe je zelf de notificaties op Github hebt geconfigureerd. Dat kan je zelf bekijken/instellen op deze manier:
![How to configure notifications](https://user-images.githubusercontent.com/1159672/203512534-cf49d8aa-f1d1-4feb-816c-9d1fd30030e2.gif)
